src/coprocessor_v2/endpoint.rs

```diff
@@ -23,9 +24,17 @@ pub struct Endpoint {
 impl tikv_util::AssertSend for Endpoint {}
 
 impl Endpoint {
-    pub fn new() -> Self {
+    pub fn new(copr_cfg: &Config) -> Self {
+        let mut plugin_registry = PluginRegistry::new();
+
+        std::fs::create_dir_all(&copr_cfg.coprocessor_plugin_directory)
+            .expect("Unable to create directory for coprocessor plugins.");
+        plugin_registry
+            .start_hot_reloading(&copr_cfg.coprocessor_plugin_directory)
+            .expect("Unable to load coprocessor plugins from directory.");
```

Comment:
What should we do if we can't load plugins or create the directory? Just let it crash? Log it?

Reply:
I'd like to log an error. 😄

Reply:
Done, PTAL :blush: 
Also moved `std::fs::create_dir_all(&plugin_directory)?;` into `PluginRegistry::start_hot_reloading()`.